http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/internal/NettyConfiguration.java

```diff
@@ -34,6 +34,7 @@
 public final class NettyConfiguration {
     private static final Duration DEFAULT_READ_TIMEOUT = Duration.ofSeconds(60);
     private static final Duration DEFAULT_WRITE_TIMEOUT = Duration.ofSeconds(60);
+    private static final Integer MAX_PENDING_ACQUIRE =10_000;
```

Comment:
Extra space. Does checkstyle not catch this?

Reply:
Ah, forgot to run the mvn command.